bagitobjecttransfer/recordtransfer/emails.py

```diff
@@ -109,7 +109,7 @@ def send_thank_you_for_your_transfer(form_data: dict, submission: Submission):
             subject="Thank You For Your Transfer",
             template_name="recordtransfer/email/transfer_success.html",
             context={
-                "archivist_email": settings.ARCHIVIST_EMAIL,
+                "archivist_email": bagitobjecttransfer.settings.base.ARCHIVIST_EMAIL,
```

Comment:
If you import settings from django.conf, you can just do this:

```suggestion
                "archivist_email": settings.ARCHIVIST_EMAIL,
```